power_python/tests/test_power_bank.py

```diff
@@ -0,0 +1,77 @@
+import pytest
+
+from ..power_source import PowerSource
+from ..power_consumer import PowerConsumer
+from ..power_bank import PowerBank
+
+def test_init():
+    pb = PowerBank(100, 200, 2000)
+
+    assert isinstance(pb, PowerBank)
+
+    with pytest.raises(TypeError):
+        assert PowerBank()  # pylint: disable=no-value-for-parameter
+
+def test_capacity():
+    pb = PowerBank(100, 200, 2000)
+    capacity = 2000
+    capacity_different = 3000
+
+    assert pb.capacity() == capacity
+    assert pb.capacity() != str(capacity)
+    assert pb.capacity() != capacity_different
+
+def test_stored_power():
+    pb = PowerBank(100, 200, 2000)
+    
+    assert pb.stored_power() == 0
+
+def test_output():
+    pb = PowerBank(100, 200, 2000)
+
+    assert pb.output() == 0
+
+def test_max_output():
+    pb = PowerBank(100, 200, 2000)
+
+    assert pb.max_output() == 200
+
+def test_connect():
+    pb = PowerBank(100, 200, 2000)
+    pb_2 = PowerBank(100, 200, 2000)
+
+    assert not pb.connect(pb)
+    assert pb.connect(pb_2)
```

Comment:
Según la documentación, tampoco debería ser posible conectar un mismo objeto más de una vez, por lo que hubiera esperado también una prueba:
```python
assert pb.connect(pb_2)
assert not pb.connect(pb_2)
```

Reply:
Adicionalmente, hubiera sido interesante algo como:
```python
assert pb.connect(pb_2)
assert pb_2.connect(pb) # o quizás not? podrían dos baterías conectarse entre sí como emisoras y receptoras de electricidad?
```

Reply:
Lo segundo no se me ocurrio, pero para lo primero hice un test similar en [power_source](https://github.com/didierrenardub/parcial_testing_2021/pull/1/files#diff-2b546e27e89771601105f6067a35be22fb4c255cbafc25f9543bf5b6321aa7f7R29) 😄

Reply:
Ahhh, no lo noté, mala mía!